src/rpc/blockchain.cpp

```diff
@@ -1605,8 +1607,18 @@ static RPCHelpMan getdeploymentinfo()
             LOCK(cs_main);
             CChainState& active_chainstate = chainman.ActiveChainstate();
 
-            const CBlockIndex* tip = active_chainstate.m_chain.Tip();
-            CHECK_NONFATAL(tip);
+            const CBlockIndex* tip;
```

Comment:
7f15c1841b98de6931a7ac68e16635a05d3e96cf: this variable should probably be called `block`. I suggest also renaming the `tip` parameter of `DeploymentInfo(` and the `active_chain_tip` parameter of `SoftForkDescPushBack ` to `block`

Reply:
Agree, but leaving this for later